user/libgnomekbd/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "libgnomekbd"
+pkgver = "3.28.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dintrospection=false"]
+hostmakedepends = [
+    "gettext",
+    "glib-devel",
+    "gobject-introspection",
+    "meson",
+    "pkgconf",
+]
+makedepends = ["gtk+3-devel", "libxklavier-devel"]
+pkgdesc = "Gnome keyboard configuration library"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://gitlab.gnome.org/GNOME/libgnomekbd"
+source = f"$(GNOME_SITE)/libgnomekbd/{pkgver[:-2]}/libgnomekbd-{pkgver}.tar.xz"
+sha256 = "22dc59566d73c0065350f5a97340e62ecc7b08c4df19183804bb8be24c8fe870"
+# needs graphical environment
+options = ["!check"]
```

Comment:
for the checks that need graphical environment, `xserver-xorg-xvfb` in `checkdepends` and `make_check_wrapper = ["xvfb-run"]` should do the job (we usually prefer `wlheadless-run`  but gtk+3 has trouble with headless wayland)

Reply:
libgnomekbd doesn't have tests, it just says "No tests defined."

Reply:
Meson does compile tests, but it doesn't run them.

Reply:
if the check phases passes without issues then remove the !check even if no tests are ran

Reply:
yeah, and i'm asking you to run them if possible, as i already explained